src/spec/junit_formatter.cr

```diff
@@ -18,7 +18,7 @@ module Spec
       io = @io
       io.puts %(<?xml version="1.0"?>)
       io << %(<testsuite tests=") << @results.size
-      io << %(" disabled=") << (@summary[:pending]? || 0)
+      io << %(" skipped=") << (@summary[:pending]? || 0)
```

Comment:
Where's the specification for this format? Are we sure the rest of the formatter is up to spec>

Reply:
I was using sources below as the reference:
- https://help.catchsoftware.com/display/ET/JUnit+Format
- https://stackoverflow.com/questions/4922867/what-is-the-junit-xml-format-specification-that-hudson-supports

Reply:
Note that, before this PR `pending` tests were simply ignored. I chose to go with `skipped` because that's the only option for `<testcase>` besides `<error/>` and `<failure/>`.